### PR TITLE
Fix: Add support for 4.2 version

### PR DESCRIPTION
**Pull request by MISTikus**
I guess this is all we need to do...
Debug successfuly started and working fine...

* Fix of #34

### CODE REVIEW

File: src/GitExtensions.SolutionRunner/GitExtensions.SolutionRunner.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net6.0-windows</TargetFramework>
-    <VersionPrefix>6.0.2</VersionPrefix>
+    <VersionPrefix>6.0.3</VersionPrefix>
```
Comment (maraf) [inline src/GitExtensions.SolutionRunner/GitExtensions.SolutionRunner.csproj]:
```suggestion
    <VersionPrefix>7.0.0</VersionPrefix>
```

**Comment by maraf** [comment]
Thank you!

**Comment by MISTikus** [comment]
@maraf, should I up version to 7.0?

**Comment by maraf** [comment]
> @maraf, should I up version to 7.0?

I already did it

**Comment by maraf** [comment]
I'll publish v7 later today

**Pull request merged**